test/common/router/config_impl_test.cc

```diff
@@ -5439,6 +5466,19 @@ name: foo
   EXPECT_EQ(1, route3->tracingConfig()->getRandomSampling().denominator());
   EXPECT_EQ(3, route3->tracingConfig()->getOverallSampling().numerator());
   EXPECT_EQ(0, route3->tracingConfig()->getOverallSampling().denominator());
+
+  const Tracing::CustomTagMap& custom_tags = route3->tracingConfig()->getCustomTags();
+  std::vector<std::string> custom_tag_views;
+  for (const auto& it : custom_tags) {
+    custom_tag_views.emplace_back(
+        dynamic_cast<const Tracing::GeneralCustomTag*>(it.second.get())->toString());
```

Comment:
Per other comment, please figure out a different way to test this, optimally by actually using the config to create spans and verify them.